CI: flatpak build fails on lack of pip

Steps to reproduce
------------------
Push something (working) to master or a branch


Expected Output
---------------
No errors

Actual Output
-------------
```
Run ./.ci/flatpak-test.sh
/usr/bin/python3: No module named pip
Error: Process completed with exit code 1.
```

Example builds
---------------------

* https://github.com/quodlibet/quodlibet/runs/1235914975
* https://github.com/quodlibet/quodlibet/runs/1235609780?check_suite_focus=true

--- a/.ci/flatpak-install.sh
+++ b/.ci/flatpak-install.sh
@@ -9,3 +9,4 @@ sudo apt-get install -y ca-certificates flatpak xvfb
 flatpak remote-add --user flathub https://dl.flathub.org/repo/flathub.flatpakrepo
 flatpak remote-add --user gnome-nightly https://nightly.gnome.org/gnome-nightly.flatpakrepo
 flatpak install --user -y flathub io.github.quodlibet.QuodLibet
+flatpak install --user -y flathub org.gnome.Sdk//3.38

--- a/.ci/flatpak-test.sh
+++ b/.ci/flatpak-test.sh
@@ -8,5 +8,5 @@ if [[ $1 == "inflatpak" ]]; then
     python3 -m pip install --user pytest flake8
     python3 setup.py test
 else
-    xvfb-run -a flatpak run --user --command="bash" io.github.quodlibet.QuodLibet "${DIR}"/flatpak-test.sh inflatpak
+    xvfb-run -a flatpak run --devel --user --command="bash" io.github.quodlibet.QuodLibet "${DIR}"/flatpak-test.sh inflatpak
 fi

--- a/.github/workflows/main.yml
+++ b/.github/workflows/main.yml
@@ -56,8 +56,6 @@ jobs:
           MSYSTEM= python3 -R -bb setup.py test
 
   flatpak:
-    # Disable until this is fixed - see #3469
-    if: false
     runs-on: ubuntu-20.04
     steps:
       - name: Checkout code

